packages/tasit-sdk/src/ethers.test.js

```diff
@@ -0,0 +1,164 @@
+      "62eb7b";
+
+    expect(signedTx).to.be.equals(expectedSignedTx);
+  });
+
+  it("should broadcast signed tx", async function() {
+    rawTx.nonce = await ropstenProvider.getTransactionCount(wallet.address);
+    let signedTx = await wallet.sign(rawTx);
+    let sentTx = await ropstenProvider.sendTransaction(signedTx);
+    expect(sentTx).to.be.not.undefined;
+
+    await ropstenProvider.waitForTransaction(sentTx.hash);
+    let txResponse = await ropstenProvider.getTransaction(sentTx.hash);
+    expect(txResponse.blockHash).to.be.not.undefined;
+  });
+
+  it("should sign a message", async function() {
+    const rawMsg = "Hello World!";
+    let signedMsg = await wallet.signMessage(rawMsg);
+
+    const expectedSignedMsg =
+      "0xcdd313d56ac1945e799a8c9d28d971fe40231ab354178611fbfe9b5ec00f493a4e33" +
+      "2781e2e72c1978b0d5540ff9fb667c54dc5c0792d9cf9b0ec10485da0d021b";
+
+    expect(signedMsg).to.be.equals(expectedSignedMsg);
+  });
+
+  it("should sign a binary message", async function() {
+    // The 66 character hex string MUST be converted to a 32-byte array first!
+    const hash =
+      "0x3ea2f1d0abf3fc66cf29eebb70cbd4e7fe762ef8a09bcc06c8edf641230afec0";
+
+    let binData = ethers.utils.arrayify(hash);
+
+    const expectedSignedBinData =
+      "0x4d6e6ed24078bbdb3e2d45a01276666c64f396915050271cc40e5874448b5e876ea6" +
+      "44cb1887e70cc6acd0cf6bfc13146261748c5450d8e21f8980ac7defda811b";
+
+    let signedBinData = await wallet.signMessage(binData);
+
+    expect(signedBinData).to.be.equals(expectedSignedBinData);
+  });
+
+  it("should resolve ENS name", async function() {
+    let address = await ensSample.provider.resolveName(ensSample.name);
+    expect(address).to.be.equals(ensSample.address);
+  });
+
+  it("should lookup ENS address", async function() {
+    let name = await ensSample.provider.lookupAddress(ensSample.address);
+    expect(name).to.be.equals(ensSample.name);
+  });
+
+  it("should instantiate a contract object", async function() {
+    contract = new ethers.Contract(contractAddress, contractABI, wallet);
+    expect(contract.address).to.be.equals(contractAddress);
+  });
+
+  it("should get/set contract's value", async function() {
+    let currentValue = await contract.getValue();
+    let message = `I like dogs ${randomWallet.mnemonic}`;
+    expect(currentValue).to.be.not.equals(message);
+
+    let updateValueTx = await contract.setValue(message);
+    await ropstenProvider.waitForTransaction(updateValueTx.hash);
+
+    let newValue = await contract.getValue();
+
+    expect(newValue).to.be.equals(message);
+  });
+
+  it("should watch contract's ValueChanged event", async function() {
+    let oldValue = await contract.getValue();
+    let newValue = `I like cats ${randomWallet.mnemonic}`;
+
+    let tx = await contract.setValue(newValue);
+
+    await waitForEvent("ValueChanged", [wallet.address, oldValue, newValue]);
```

Comment:
@marcelomorgado I miss a descriptive line like this from the ethers.js assertions we used to have in the app itself https://github.com/tasitlabs/tasit/pull/10/files#diff-e368a8867dce79d5fe6876e3679012dcL197